OrbitQt/TutorialOverlay.h

```diff
@@ -10,6 +10,7 @@
 #include <QPoint>
 #include <memory>
 #include <optional>
+#include <functional>
```

Comment:
clang-format I guess.

Reply:
Yessss! I should go back to primary school and learn the alphabet. Thanks, fixed!